templates/web/base/waste/bulky/summary.html

```diff
@@ -1,28 +1,157 @@
-%]
-
-[% BLOCK answers %]
-  [% FOR item_key IN data.keys.grep('^item').sort %]
-    [% item = data.$item_key; NEXT UNLESS item %]
-    <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key govuk-summary-list__key--sub">[% item_key.ucfirst.replace( '_', ' ' )  %]</dt>
-        <dd class="govuk-summary-list__value">
-          [% item.item %]
-          <br>
-          [% extra_text = form.items_extra_text.${item.item}; IF extra_text %]
-            Notes: [% extra_text %]
-            <br>
-          [% END %]
-          [% IF item.photo_fileid %]
-          <img src="/photo/temp.[% item.photo_fileid %]" />
+[% USE date(format = c.cobrand.bin_day_format) %]
+[% PROCESS 'waste/header.html' %]
+[% PROCESS 'govuk/fields.html' ~%]
+
+[% PROCESS errors %]
+[% SET data = form.saved_data ~%]
+
+
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-9">Request a bulky waste collection</h1>
+<div class="bulky__summary">
+  <h2 class="govuk-heading-l govuk-!-margin-bottom-5">Booking Summary</h2>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-warning-text due">
+        <div class="govuk-warning-text__img">
+          <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        </div>
+        <div class="govuk-warning-text__content">
+            <span class="govuk-warning-text__assistive">Warning</span>
+              <p class="govuk-!-margin-bottom-0">Please read carefully all the details for your booking. You won't be
+                able to amend your details afterwards.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h3>Property details</h3>
+      [% INCLUDE 'waste/_address_display_bulky_summary.html' %]
+
+      <hr>
+      <h3>Collection date</h3>
+      <dl>
+          <dt>Date</dt>
+          <dd>[% cobrand.bulky_nice_collection_date(data.chosen_date) %]</dd>
+      </dl>
+
+      <div class="govuk-warning-text due">
+        <div class="govuk-warning-text__img">
+          <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        </div>
+        <div class="govuk-warning-text__content">
+            <span class="govuk-warning-text__assistive">Warning</span>
+            <p class="govuk-!-margin-bottom-3">You can cancel this booking till
+              23:55hrs on the 2nd of August 2022. [% #XXX calculate cut off when we do cancellations  %]</p>
+            <p class="govuk-!-margin-bottom-0">You can get a refund by cancelling 24hrs
+               before the day of collection.</p>
+        </div>
+      </div>
+
+      <hr>
+      <h3>Items to be collected</h3>
+      [% items = [] %]
+      [% FOR item_key IN data.keys.grep('^item_[\d]+$').sort %]
+        [% item = data.$item_key; NEXT UNLESS item %]
+        [% items.push(item) %]
+      [% END %]
+      <dl>
+        <dt>Price</dt>
+        <dd>£23.50 [% #XXX %]</dd>
```

Comment:
If payment branch is merged and this rebased on top, can presumably make payment info available here, there's now a function to work it out (can't recall if it'll be already present for this template).